py/occampy3/wrappers/sbm_manager.py

```diff
@@ -17,17 +17,20 @@ def __init__(self, ref=None):
     def make_model(self, model_name: str, make_project: bool) -> Model:
         return Model(self._ref.makeSbModel(model_name, make_project))
 
-    def getTestSampleSz(self) -> double:
+    @property
+    def test_sample_size(self) -> double:
         return self._ref.getTestSampleSize()

```

Comment:
Still have the name mixed up on this one, the pyoccam SBMManager class function name is getTestSampleSz.